### PR TITLE
Ofmcc 1399 split classroom flag

### PR DESCRIPTION
**Pull request by jgstorey**

### CODE REVIEW

File: frontend/src/components/ui/AppYesNoInput.vue [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    emitValue() {
+      this.$emit('input', this.internalSelection);
+    },
+  },
+};
+</script>
+
+<style scoped>
+.yes-no-button.v-btn--active {
+  background-color: #003366 !important;
```
[the remark this answers is not in the document]
Comment (weskubo-cgi) [reply frontend/src/components/ui/AppYesNoInput.vue]: Removed and added additional style to show border.

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/components/licences/LicenceDetails.vue [hunk 2]
```diff
@@ -124,6 +124,42 @@
...
+                        <v-tooltip content-class="tooltip" :text="SPLIT_CLASSROOM_INFO_TXT" top>
+                          <template v-slot:activator="{ props }">
+                            <v-icon size="large" v-bind="props">mdi-information-slab-circle-outline</v-icon>
+                          </template>
+                        </v-tooltip>
+                        :
+                      </AppLabel>
+                    </v-col>
+                    <v-col cols="12" sm="4" lg="3" xl="3" class="pl-4">
+                      <AppYesNoInput v-model="licenceDetail.applyRoomSplitCondition" :disabled="readOnly"></AppYesNoInput>
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/components/licences/LicenceDetails.vue:141]: Fixed.

File: frontend/src/components/licences/LicenceDetails.vue [hunk 3]
```diff
@@ -166,9 +207,17 @@ export default {
       return healthAuthority ? healthAuthority : this.BLANK_FIELD
     },
   },
+  watch: {
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/components/licences/LicenceDetails.vue:210]: Removed, a hold over from when we were headed for previous licence updates.

File: frontend/src/components/licences/LicenceDetails.vue [hunk 4]
```diff
@@ -166,9 +207,17 @@ export default {
...
+    requiresSplitClassroom: {
+      handler(newVal) {
+        console.log('requiresSplitClassroom', newVal)
+      },
+    },
+  },
   async created() {
     this.panel = this.allLicenceDetailsID
     this.BLANK_FIELD = '- - - -'
+    this.SPLIT_CLASSROOM_INFO_TXT = 'This is a placeholder message'
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/components/licences/LicenceDetails.vue:220]: Fixed.

File: frontend/src/components/ui/AppYesNoInput.vue [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+      class="yes-no-button">
+      N
+    </v-btn>
+  </v-btn-toggle>
+</template>
+
+<script>
+export default {
+  name: 'AppYesNoInput',
+  props: ['value'], // Expect a boolean value from the parent
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/components/ui/AppYesNoInput.vue:11]: Fixed.

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**